awswrangler/distributed/_pool.py

```diff
@@ -0,0 +1,23 @@
+"""Threading Module (PRIVATE)."""
+
+import itertools
+import logging
+from typing import Any, Callable, List, Optional, Union
+
+import boto3
+from ray.util.multiprocessing import Pool
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+class _RayPoolExecutor:
+    def __init__(self, processes: Optional[Union[bool, int]] = None):
+        self._exec: Pool = Pool(processes=None if isinstance(processes, bool) else processes)
```

Comment:
1. Yes - this is due to broader topic of whether we should re-use `use_threads` arg to both configure number of threads in multithreaded single-node impl and ray parallelism
2. The default is `auto` which connects to whatever cluster that's already been initialized since we're running `ray.init` on import it's not needed

Reply:
Ah wait actually that's a no on point 1 - if `processes` is an int it will be passed to the Pool. But yes it's still part of broaded dicsussion of whether we reuse existing arguments to configure ray (or, for example, only use config) to pass them